browser/themes/brave_theme_service.cc

```diff
@@ -23,7 +23,7 @@ const ThemeHelper& GetBraveThemeHelper(Profile* profile) {
 #endif
   // Because the helper is created as a NoDestructor static, we need separate
   // instances for regular vs tor/guest profiles.
-  if (brave::IsTorProfile(profile) || brave::IsGuestProfile(profile)) {
+  if (profile->IsTor() || brave::IsGuestProfile(profile)) {
```

Comment:
Hmm, I think I gave a wrong review here.
This returns always false even if current window is tor because ThemeService uses redirected profile.
Fortunately, this will not make any regression to theme service becaue tor profile is also incognito now and incognito and tor uses same theme.
However, I'm not sure other areas have this issue or not.

But my another PR(https://github.com/brave/brave-core/pull/6912) needs more work because I need to use different theme colors for private and tor window. :)